### PR TITLE
Test gentoo support

### PR DESCRIPTION
**Pull request by Petross404**
I am not sure why your commits appear to my PR, but it's supposed to be "able to merge".

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -1,12 +1,17 @@
 cmake_minimum_required(VERSION 3.1)
 project(gpio++ VERSION 4.1)
 
+include(GNUInstallDirs)
+
+find_library(asio REQUIERED FATAL)
```
Comment (dimitry-ishenko) [inline CMakeLists.txt]: You have a typo in `REQUIRED`.

File: CMakeLists.txt [hunk 2]
```diff
@@ -16,15 +21,20 @@ set_property(TARGET gpio++-base PROPERTY POSITION_INDEPENDENT_CODE ON)
...
+endif()
 
 ########################
 # install
-install(DIRECTORY include/ DESTINATION include)
-install(FILES ${FILES} DESTINATION share/doc/${PROJECT_NAME}-${VERSION})
+install(DIRECTORY include/ DESTINATION ${CMAKE_INSTALL_INCLUDEDIR})
+install(FILES ${FILES} DESTINATION ${CMAKE_INSTALL_DOCDIR})
+#install(DIRECTORY include/ DESTINATION include)
+#install(FILES ${FILES} DESTINATION share/doc/${PROJECT_NAME}-${VERSION})
```
Comment (dimitry-ishenko) [inline CMakeLists.txt]: Please don't comment the lines. Just take them out.

**Comment by dimitry-ishenko** [comment]
Please rebase your repo to get rid of the two commits at the top.

File: CMakeLists.txt (hunk 1 above)
Comment (dimitry-ishenko) [reply CMakeLists.txt]: Also in order for `find_library` to work, you need `FindAsio.cmake` module, which needs to be written...

Maybe for now just add a dependency in your ebuild and I will try to work it some time later.

File: CMakeLists.txt (hunk 2 above)
Comment (Petross404) [reply CMakeLists.txt]: I totally forgot them

**Comment by dimitry-ishenko** [comment]
Please test version 4.1

**Comment by Petross404** [comment]
> Please test version 4.1

I noticed this just now. I will.

**Pull request closed without merging**